.ibm/pipelines/auth/secrets-rhdh-secrets.yaml

```diff
@@ -10,13 +10,11 @@ data:
   GITHUB_APP_CLIENT_SECRET: dGVtcA==
   GITHUB_APP_WEBHOOK_URL: aHR0cHM6Ly9zbWVlLmlvL0NrRUNLYVgwNzhyZVhobEpEVzA=
   GITHUB_APP_WEBHOOK_SECRET: dGVtcA==
-  GITHUB_ENABLED: dHJ1ZQ==
```

Comment:
@subhashkhileri why are you deleting this secret ?

Reply:
@gustavolira this is longer needed in 1.0+ version